### PR TITLE
Enhance developing process

### PR DESCRIPTION
**Pull request by nickhsine**
For better developing experience,
I refactor the process we start up the keystone server.

Please have a see if available.

### CODE REVIEW

**Comment by nickhsine** [comment]
For [[draft] integration with CircleCI](https://github.com/twreporter/plate/pull/108/commits/469e837306d722e33636efaecb8a861cb34ac528) commit, it doesn't rollout new docker image on pods automatically.
The reason is that `keystone-plate` server doesn't support scalability right now.
If we scale the servers while rolling updating, it will cause requests forbidden problems.

File: Dockerfile [hunk 1]
```diff
@@ -1,44 +1,20 @@
...
 
-# Copy source files
-COPY . $REACT_SOURCE
-RUN cd $REACT_SOURCE
-RUN npm install 
-
-# Set time zone
-RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+# Add built source files
+COPY . ./
```
Comment (babygoat) [inline Dockerfile]: Could you ensure that the container from the image could successfully loading all the static assets?

File: .dockerignore [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+*.txt
+*.png
+Makefile
+makefile
+package.json
+yarn.lock
+LICENSE
+Dockerfile
+build-config.js
+Procfile
```
Comment (babygoat) [inline .dockerignore]: I suggest to remove Procfile completely as we don't have to deploy on Heroku

File: Dockerfile [hunk 3]
```diff
@@ -1,44 +1,20 @@
-FROM node:4.4-slim
+FROM node:10.15-slim
 
-RUN groupadd user && useradd --create-home --home-dir /home/user -g user user
+ARG app_group=main_site_user
+ARG app_user=main_site_user
```
Comment (babygoat) [inline Dockerfile]: rename user, group to keystone_user

File: Dockerfile [hunk 4]
```diff
@@ -1,44 +1,20 @@
...
-# Copy source files
-COPY . $REACT_SOURCE
-RUN cd $REACT_SOURCE
-RUN npm install 
-
-# Set time zone
-RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+# Add built source files
+COPY . ./
+COPY config-for-docker-build.js ./config.js
```
Comment (babygoat) [inline Dockerfile]: config-for-docker-build.js already exist.
use
`RUN mv config-for-dockerbuild.js config.js`

**Pull request merged**